### DISCUSSION TITLE
Improve wiki "Where are sources"

### DISCUSSION THREAD

**Discussion by thernstig** [stm32duino/Arduino_Core_STM32#2562] [WiKi] [answered]
Reference: https://github.com/stm32duino/Arduino_Core_STM32/wiki/Where-are-sources

The wiki says:

> The location is displayed in the "Preferences" dialog.

This is not true. The preferences dialog contains the location for where the Sketchers are saved, but not where the installation of the various packages/platforms exist.

See here:

![image](https://github.com/user-attachments/assets/04353e13-7e0a-45bd-883b-8da2cffac246)

In addition, the wiki should add `Windows 11` to this part:

```text
c:\Users\\<USERNAME\>\AppData\Local\Arduino15\
```

Actually, you should just mention Windows 10 and Windows 11 and those locations, as previous version are not supported for Arduino 2.0, see here https://forum.arduino.cc/t/ide-2-2-1-compatibility-to-windows-8-1-64-bit/1201679.

**Comment by thernstig** [comment]
@fpistm unsure why this was converted into a discussion, as the Wiki is incorrect?

**Comment by fpistm** [comment]
Hi @thernstig 

I convert it as a discussion in a dedicated section for the WiKi as not related to issue (bug) with the core itself.

You are right, it was written when only Arduino 1.8.x existed. It  requires update for the Arduino IDE 2.x. Any contribution are welcome.

**Reply by thernstig** [reply]
I will do a contribution tomorrow.

(Of course you can decide what should constitute issues in this repo as a maintainer 🙂 It is however very idiomatic for repositories to let documentation faults/inconsistencies, including wikis, to exist as issues. The problem is otherwise that users that are accustomed to searching for issues might miss them if they reside as discussions).

**Reply by fpistm** [reply]
GitHub search also in the discussion so not an issue.
Before, all user can edit/create WiKi page but several times some spam, advertising, ... were added, that's why I had to forbid public update.
Anyway I though it is possible to create PR to update it.

**Reply by thernstig** [reply, marked as answer]
@fpistm I gave it a try here: https://github.com/thernstig/Arduino_Core_STM32/wiki/Where-are-sources

**Reply by fpistm** [reply]
I've merged your change and made a small update tor remove the portable IDE section which is not supported by Arduino IDE 2.
